docs/core/compatibility/xml/7.0/xmlsecureresolver-obsolete.md

```diff
@@ -0,0 +1,68 @@
+---
+title: ".NET 7 breaking change: XmlSecureResolver is obsolete"
+description: Learn about the .NET 7 breaking change in XML/XSLT where XmlSecureResolver was obsoleted and XmlSecureResolver.GetEntity unconditionally throws a run-time exception.
+ms.date: 09/08/2022
+---
+# XmlSecureResolver is obsolete
+
+The method <xref:System.Xml.XmlSecureResolver.GetEntity(System.Uri,System.String,System.Type)?displayProperty=fullName> unconditionally throws an <xref:System.Xml.XmlException> at run time. If your application utilizes <xref:System.Xml.XmlSecureResolver> and you attempt to resolve an entity through it, entity resolution will fail with an exception.
+
+Additionally, the entirety of the type <xref:System.Xml.XmlSecureResolver?displayProperty=fullName> is obsolete as warning. All references to this type will result in a [SYSLIB0047](../../../../fundamentals/syslib-diagnostics/syslib0047.md) warning at build time. If you've enabled warnings as errors, this will cause a build break if your application references <xref:System.Xml.XmlSecureResolver>.
```

Comment:
```suggestion
Additionally, the entire <xref:System.Xml.XmlSecureResolver?displayProperty=fullName> type is obsolete. All references to this type will result in a [SYSLIB0047](../../../../fundamentals/syslib-diagnostics/syslib0047.md) warning at build time. If you've enabled warnings as errors, this will cause a build break if your application references <xref:System.Xml.XmlSecureResolver>.
```